Source/Core/DiscIO/DirectoryBlob.cpp

```diff
@@ -730,34 +741,33 @@ void DirectoryBlobReader::SetPartitionHeader(DirectoryBlobPartition* partition,
   const u64 max_cert_size = H3_OFFSET - cert_offset;
 
   u64 cert_size;
-  if (wrapped_partition)
+  if (wrapped)
   {
-    const auto& cert = m_wrapped_volume->GetCertificateChain(*wrapped_partition);
-    auto& new_cert = m_extra_data.emplace_back(cert);
+    std::vector<u8> new_cert = std::move(volume->GetCertificateChain(*wrapped_partition));
```

Comment:
Unnecessary `std::move`. (You already have an rvalue.)

```suggestion
    std::vector<u8> new_cert = volume->GetCertificateChain(*wrapped_partition);
```